packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -306,4 +306,298 @@
     </SF.Item>
   </Shw::Flex>
 
+  <Shw::Text::H3>Checkmark</Shw::Text::H3>
+  <Shw::Flex as |SF|>
+    <SF.Item>
```

Comment:
I would add `@label` declarations to these (and other) items, is really hard to understand what they "showcase"

Reply:
Good suggestion – added